src/NBitcoin/Network.cs

```diff
@@ -64,6 +64,14 @@ public abstract class Network
     {
         protected Block Genesis;
 
+        /// <summary>
+        /// The default amount of seconds to keep misbehaving peers from reconnecting.
+        /// <para>
+        /// This value should be calculated as (TargetSpacingSeconds * maxReorgLength) / 2.
+        /// </para>
+        /// </summary>
+        public int BanTimeSeconds { get; protected set; }
```

Comment:
`DefaultBanTimeSeconds`

Reply:
Change the name to prevent misuse of the default value when it may  have been overridden from the command line.

Reply:
Done.